src/service/vsync-impl.js

```diff
@@ -441,14 +441,19 @@ function callTaskNoInline(callback, state) {
   return true;
 }
 
-
 /**
  * @param {!Window} window
  * @return {!Vsync}
  */
+export function getVsyncService(window) {
+  installVsyncService(window);
+  return getService(window, 'vsync');
+}
```

Comment:
Sorry I meant we should just use `vsyncFor` instead of creating another method. `vsync` should be installed in `runtime.js` right?